lib/shared/widgets/givt_tertiary_elevated_button.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:givt_app_kids_web/shared/widgets/givt_elevated_button.dart';
+
+class GivtTertiaryElevatedButton extends StatelessWidget {
+  const GivtTertiaryElevatedButton({
+    required this.text,
+    required this.onPressed,
+    this.isLoading = false,
+    super.key,
+  });
+
+  static const Color _tertiaryBackgroundColor = Color(0xFFE28D4D);
```

Comment:
Wouldn't it be better to have those colors in one separated file?

Reply:
yes, initially I thought to create apptheme file (as in givt app) and move them there. 
However I decided to wait until we will have more clarity about the themeing/colors approach we are working on at the moment with Elmar. So the code is prepared for such change soon.